<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>card_status</class>
 <widget class="QDialog" name="card_status">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>640</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>更纱黑体 SC</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>校园卡状态管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#card_status{
background-color: rgb(255, 255, 255);
border-image: url(:/image/image/backpage_common.png);
}</string>
  </property>
  <widget class="QLineEdit" name="line_stu_id">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="placeholderText">
    <string>请输入学号</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_id">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);</string>
   </property>
   <property name="maxLength">
    <number>18</number>
   </property>
   <property name="placeholderText">
    <string>学生身份证号</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="placeholderText">
    <string>学生姓名</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_getCard">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_getCard{

border:1px groove gray;
border-radius:5px;
padding:2px 4px;

color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 128);
}

QPushButton#btn_getCard:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,77);
}
 
QPushButton#btn_getCard:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 144);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_getCard:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>办卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reportLoss">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_reportLoss{

border:1px groove gray;
border-radius:5px;
padding:2px 4px;

color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 128);
}

QPushButton#btn_reportLoss:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,77);
}
 
QPushButton#btn_reportLoss:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 144);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_reportLoss:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>挂失</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelReportLoss">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_cancelReportLoss{

border:1px groove gray;
border-radius:5px;
padding:2px 4px;

color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 128);
}

QPushButton#btn_cancelReportLoss:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,77);
}
 
QPushButton#btn_cancelReportLoss:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 144);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_cancelReportLoss:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>解挂失</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_makeUpCard">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_makeUpCard{

border:1px groove gray;
border-radius:5px;
padding:2px 4px;

color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 128);
}

QPushButton#btn_makeUpCard:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,77);
}
 
QPushButton#btn_makeUpCard:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 144);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_makeUpCard:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>补卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_logoutCard">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_logoutCard{

border:1px groove gray;
border-radius:5px;
padding:2px 4px;

color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 128);
}

QPushButton#btn_logoutCard:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,77);
}
 
QPushButton#btn_logoutCard:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 144);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_logoutCard:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
